wolfcrypt/benchmark/benchmark.c

```diff
@@ -2412,8 +2424,12 @@ static void* benchmarks_do(void* args)
         !defined(NO_HW_BENCH)
         bench_aes_aad_options_wrap(bench_aesgcm, 1);
     #endif
-
-        bench_gmac();
+        #ifndef NO_SW_BENCH
```

Comment:
Spacing - this line needs to back up one tab

Reply:
Thanks, fixed.